src/main/java/core/basesyntax/Loterry.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Loterry {
+    public static final ColorSupplier colorSupplier = new ColorSupplier();
+    public static final Random random = new Random();
+    public static final int UPPER_BOUND = 100;
+
+    public Ball getRandomBall() {
+        String color = String.valueOf(colorSupplier.getRandomColor());
```

Comment:
String.valueOf calls toString under the hood. You need to call .name(), as was mentioned in the checklist